src/main/java/org/opentripplanner/transit/model/network/Route.java

```diff
@@ -55,7 +56,7 @@ public final class Route extends AbstractTransitEntity<Route, RouteBuilder> impl
     this.branding = builder.getBranding();
     this.groupsOfRoutes = listOfNullSafe(builder.getGroupsOfRoutes());
     this.gtfsType = builder.getGtfsType();
-    this.gtfsSortOrder = builder.getGtfsSortOrder();
+    this.gtfsSortOrder = IntUtils.requireNullOrNotNegative(builder.getGtfsSortOrder(), "sortOrder");
```

Comment:
What happens if this is negative? The whole route is ignored? Does OBA validate this also?

Reply:
Since this is one of the least important fields, would it be possible to put a message in the build report instead of failing the build?

Reply:
It would be possible but I'm personally trying to become stricter about not accepting poor input data. We regularly debate what is the best strategy here but I'm not sure we have settled on a position.